src/app/dashboard/workshop/collection/collection.component.ts

```diff
@@ -101,7 +101,7 @@ export class CollectionComponent extends CollectionSubscriber implements OnInit
 
   onGuildSubscribe(guild: PartialGuild) {
     this.doGuildSubscribe(guild).subscribe(resp => {
-      if (resp.success && this.guildContext.id === guild.id) {
+      if (resp.success && this.guildContext && this.guildContext.id === guild.id) {
```

Comment:
```suggestion
      if (resp.success && this.guildContext?.id === guild.id) {
```
TypeScript does null coalescing, I just have no idea what they call it.

Reply:
...I could have sworn it yelled at me for doing that before, but it seems to not anymore? 😛